docs/designs/ps_design.md

```diff
@@ -58,322 +61,180 @@ class EmbeddingTable(object):
         self.vectors = {}
```

Comment:
It seems a little strange that variable `vectors` is a python dictionary. I modified the variable name to `embeddings` in my PR. Maybe better to change a name.

Reply:
Done.  Renaming to `embedding_vectors`.